pages/_app.tsx

```diff
@@ -1,5 +1,3 @@
-import '@trussworks/react-uswds/lib/uswds.css'
-import '@trussworks/react-uswds/lib/index.css'
```

Comment:
removed these to minimize the bundle for now, we can re-add them when we set up USWDS